relay-server/src/actors/events.rs

```diff
@@ -754,6 +764,31 @@ impl EventProcessor {
         }));
     }
 
+    /// Extracts the timestamp from the minidump and uses it as the event timestamp.
+    #[cfg(feature = "processing")]
+    fn write_minidump_timestamp(event: &mut Event, minidump_item: &envelope::Item) {
+        use std::time::SystemTime;
```

Comment:
Please move the import to the top of the file. However, I think you can remove it based on a comment below.